test/invoice_item_test.rb

```diff
@@ -3,22 +3,34 @@
 require_relative '../lib/invoice_item'
 
 class InvoiceItemsTest < MiniTest::Test
+  attr_reader :parent, :invoice_item
+
+  def setup
+    sales_engine = SalesEngine.new
+    @parent = InvoiceItemRepository.new('test/support/sample_invoice_items.csv', sales_engine)
+    @invoice_item = InvoiceItems.new({:id =>"8", :item_id => "534", :invoice_id => "1", :quantity => "6", :unit_price => "76941", :created_at => "2012-03-27 14:54:09 UTC" , :updated_at => "2012-03-27 14:54:09 UTC"}, parent)
```

Comment:
Line is too long. [223/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space found before comma.<br>Space inside { missing.<br>Space inside } missing.